src/js/redux/reducers/search/autocompleteReducer.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by michaelbray on 12/12/16.
+ */
+
+const initialState = [];
+
+const autocompleteReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SET_AUTOCOMPLETE_LOCATIONS': {
+            return Object.assign([], action.locations);
```

Comment:
I'm not 100% sure you can do an `Object.assign` to clone an array. Lodash has a utility function for cloning an array. Alternatively, use an ImmutableJS structure (which we may migrate toward completely for Reducers in the future).